test/test_dgmulti_1d.jl

```diff
@@ -25,7 +25,7 @@ isdir(outdir) && rm(outdir, recursive = true)
         t = sol.t[end]
         u_ode = sol.u[end]
         du_ode = similar(u_ode)
-        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
+        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 5000
```

Comment:
```suggestion
        @test (@allocated Trixi.rhs!(du_ode, u_ode, semi, t)) < 1000
```
Maybe we can also go back to only 1000 allowed allocations now?

Reply:
It passed :+1: So now let's try to revert all these changes.